scripts/bump_version.sh

```diff
@@ -0,0 +1,33 @@
+#!/bin/bash
+
+# Setting -x is absolutely forbidden as it could leak the GitHub token.
+set -uo pipefail
+
+
+GITHUB_MAIL="${GITHUB_MAIL:-prometheus-team@googlegroups.com}"
+GITHUB_USER="${GITHUB_USER:-prombot}"
+
+# GITHUB_TOKEN required scope: repo.repo_public
+GITHUB_TOKEN="${GITHUB_TOKEN:-}"
+if [ -z "${GITHUB_TOKEN}" ]; then
```

Comment:
Minor bash nit, use `[[ ]]` for bash tests.

Reply:
done